src/game_renderer/Shape.cpp

```diff
@@ -10,51 +10,49 @@
 using namespace std;
```

Comment:
Are the changes in this file just formatting or are they functional? Im asking because I already started working on replacing it